client/checkout/classic/upe.js

```diff
@@ -213,10 +226,12 @@ jQuery( function ( $ ) {
 				} );
 				upeElement.mount( '#wcpay-upe-element' );
 				upeElement.on( 'change', ( event ) => {
+					unblockUI( $upeContainer );
```

Comment:
Is there a reason this needs to be inside the `change` handler? Could this just sit inside its containing `then` block, so it fires only once after the element is mounted instead? Or is there some timing benefit or other use for removing the placeholder whenever the user toggles the UPE payment method?

Reply:
I had initially done that as a timing thing. I think at the time, doing it outside of the event caused the loading icon to be removed too early and still left a gap. But in testing it again now, that doesn't appear to be the case! I've made that change now since there wasn't a reason to fire it with every change.